test/unit_tests/apps/owning-a-home/js/dom-tools-spec.js

```diff
@@ -35,4 +35,20 @@ describe( 'dom-tools', () => {
       );
     } );
   } );
+
+  describe( 'hasClass()', () => {
+    it( 'should return true when element has class', () => {
```

Comment:
Minor: updating the descriptions to include "the provided class" would make it clearer we're not testing if it has a class at all, I got mixed up on that when I saw the query selector for the false test was a class name.

Reply:
I omitted the `the` so it'll fit on one line :)